psychopy/hardware/eyetracker.py

```diff
@@ -79,66 +80,39 @@ def __iter__(self):
             target.units = self.units
         # Get self as dict
         asDict = {}
+
+        textColor = self.textColor
+        if isinstance(textColor, str) and textColor.lower() == 'auto':
```

Comment:
Generally, we use `None` to mean `(0, 0, 0, 0)` (transparent) with colours. I think in other stimuli I've used `False` as a way to indicate "not set" as opposed to "set as None".